### PR TITLE
Add deployer directory with hosts and recepe

### PR DESCRIPTION
**Pull request by nestordedios**

### CODE REVIEW

**Comment by anketwokings** [comment]
Am i right that I only see example files? No europeana recipe/ host.yml?

**Comment by nestordedios** [comment]
Yes! Shame on me! 

With my last commit c41a899863f41c8f85f51208e031fb7e18397e80 `hosts.yml` is in there. It was gitignored.

File: deployer/hosts.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    keep_releases: 10
+    cleanup_use_sudo: true
+    snapshots_dir: snapshots
+    shared_files: 
+      - '.bolt.yml'
+    keep_files: 
+      - 'app/config/menu.yml'
+      - 'app/config/taxonomy.yml'
+      - 'app/config/extensions/zohoimport.europeana_local.yml'
+#      - 'extensions/bundles/europeana/zohoimport/TokenStorage/zcrm_oauthtokens.txt'
```
Comment (anketwokings) [inline deployer/hosts.yml]:
```suggestion
      - 'extensions/bundles/europeana/zohoimport/TokenStorage/zcrm_oauthtokens.txt'
```
Was commented out, but I think we need it

File: deployer/deploy.php [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+        }
+    } else {
+        writeln('<info>➤</info> Not copying files along releases.');
+        foreach($keep_files as $currentfile) {
+            set('currentfile', $currentfile);
+            writeln('not doing this: cp {{current_path}}/{{currentfile}} newrelease/{{currentfile}}');
+        }
+    }
+
+    // run('chmod 777 {{release_path}}/extensions/bundles/europeana/zohoimport/TokenStorage/zcrm_oauthtokens.txt');
```
Comment (anketwokings) [inline deployer/deploy.php]:
```suggestion
    run('chmod 777 {{release_path}}/extensions/bundles/europeana/zohoimport/TokenStorage/zcrm_oauthtokens.txt');
```

File: deployer/deploy.php [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+    // run('chmod 777 {{release_path}}/vendor/zohocrm/php-sdk/src/oauth/utility/');
```
Comment (anketwokings) [inline deployer/deploy.php]:
```suggestion
     run('chmod 777 {{release_path}}/vendor/zohocrm/php-sdk/src/oauth/utility/');
```

**Review by ankedsgn — COMMENTED** [review]
I uncommented the three things I think we need. Please, someone check these. :)

--edit:ignore them, don't need those _yet_ --

**Pull request merged**